docs/api-docs/tax/tax-rates-zones.md

```diff
@@ -16,7 +16,7 @@ You can further narrow the scope of a zone by specifying one or more customer gr
 
 <!-- theme: info -->
 > #### Default tax zone
-> A default tax zone covers locations that other zones don't cover. Shoppers in customer groups that don't match a tax zone fall under the default tax zone. A default tax zone does not have `shopper_target_settings` in its requests or responses. You can't specify the customer groups or locations for a default tax zone.  
+> The default tax zone, whose zone `id` is 1, covers locations that other zones don't cover. Shoppers in customer groups that don't match a tax zone fall under the default tax zone. You cannot create, disable, or delete the default tax zone. When updating the default tax zone, you cannot adjust the locations or customer groups that it covers.  
```

Comment:
1. `whose` applies to people, not impersonal objects or idea. revise.
2. It's not just shoppers in customer groups who can be left zoneless; ungrouped customers could also have no applicable zone defined. revise.
3. You don't need to create the default tax zone, but i wouldn't put that in the same category as disabling, deleting, or mis-modifying the zone.

Reply:
@slsriehl Copyedits are made